mobile/src/libs/components/input/styles.ts

```diff
@@ -1,9 +1,51 @@
 import { StyleSheet } from 'react-native';
 
+import { AppColor } from '#libs/enums/enums';
+
 const styles = StyleSheet.create({
   input: {
     borderWidth: 1,
-    borderColor: 'gray',
+    borderColor: AppColor.GRAY_300,
+    height: 44,
+    padding: 12,
+    borderRadius: 8,
+    fontSize: 16,
+    backgroundColor: AppColor.WHITE,
+    marginBottom: 2,
+    marginTop: 6,
+  },
+  label: {
+    fontSize: 14,
+    fontWeight: 'bold',
```

Comment:
I saw you used '600' as fontWeight above. I think we should use unificated approach, but it's better to ask the coaches which option to choose